src/components/pages/OnboardPage.tsx

```diff
@@ -1,38 +1,34 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
 import Hero_Image from "@app/assets/onboard_hero_image.jpg";
 import Meshtastic_Logo from "@app/assets/Mesh_Logo_Black.png";
 import SerialPortOption from "../Onboard/SerialPortOption";
 
-type PortState = "IDLE" | "LOADING" | "SUCCESS" | "FAILURE";
+import { requestAvailablePorts } from "@features/device/deviceActions";
+import { selectAvailablePorts } from "@app/features/device/deviceSelectors";
+
+type PortState = "IDLE" | "PENDING" | "SUCCESS" | "FAILED";
```

Comment:
We're now using the connection state from the `selectRequestStateByName` call, you might not need this.

Reply:
Fixed.